src/main/java/org/jboss/ejb/client/EJBClientContext.java

```diff
@@ -73,6 +73,12 @@
 
     private static final EJBClientInterceptor.Registration[] NO_INTERCEPTORS = new EJBClientInterceptor.Registration[0];
 
+    /**
+     * Timeout defined through system property "org.jboss.ejb.received-cluster-topology-wait-time", value in property is in seconds.
+     */
+    public static final int RECEIVED_CLUSTER_TOPOLOGY_WAIT_TIMEOUT;
```

Comment:
This should not be public as that constitutes API.  If it needs to be shared between packages, perhaps a holder class in the _private package is more appropriate.